http-protocol-binding.md

```diff
@@ -429,6 +429,57 @@ Content-Length: nnnn
 
 ```
 
+### 3.4. Multipart Binary Content Mode
```

Comment:
same comment as in https://github.com/cloudevents/spec/pull/592/files#r396477050